src/library_pthread.js

```diff
@@ -571,6 +572,13 @@ var LibraryPThread = {
     else postMessage({ 'cmd': 'cleanupThread', 'thread': thread });
   },
 
+  _emscripten_thread_set_strongref: function(thread) {
+#if ENVIRONMENT_MAY_BE_NODE
+    if (!ENVIRONMENT_IS_NODE) return;
+    PThread.pthreads[thread].ref();
```

Comment:
For a two line function like this I think maybe the positive check is more readable:

```
    if (ENVIRONMENT_IS_NODE) {
       PThread.pthreads[thread].ref();
    }
```

(and maybe fewer bytes of code?)

Reply:
Addressed with 8f2bc84fc8839dead3cf46b896a3673bd7bd1578, there was no change in the code size tests.